src/org/omegat/ColorScheme_dark.properties

```diff
@@ -0,0 +1,38 @@
+# OmegaT application default colors scheme
+# This is a table of default colors definitions when no customization.
+# Here is a definition for dark theme, such as GTK LAF(dark), OSX Mojava and 3rd party dark LAFs.
+OmegaT.ActiveSource=28a028
```

Comment:
Could you normalize the color names to be `CamelCase.lowerCamelCase` please?

Reply:
done changing names.